packages/contracts/src/dollar/StakingShare.sol

```diff
@@ -1,20 +1,22 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.16;
 
-import "./ERC1155SetUri/ERC1155SetUri.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "./ERC1155SetUri/ERC1155BurnableSetUri.sol";
-import "./ERC1155SetUri/ERC1155PausableSetUri.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155URIStorage.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Burnable.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Pausable.sol";
 import "./core/UbiquityDollarManager.sol";
 import "./utils/SafeAddArray.sol";
 
 contract StakingShare is
-    ERC1155SetUri,
-    ERC1155BurnableSetUri,
-    ERC1155PausableSetUri
+    ERC1155URIStorage,
+    ERC1155Pausable,
+    ERC1155Burnable
 {
     using SafeAddArray for uint256[];
 
+    string public _uri = "";
```

Comment:
we already have this `_uri` property in the `ERC1155` contract so you can remove it

Reply:
done